src/ml/neural_net/model_spec.hpp

```diff
@@ -122,7 +125,7 @@ class model_spec {
    * \throw If a float_array's shape does not match the corresponding
    *        NeuralNetworkLayer.
    */
-  void update_params(const float_array_map& weights);
+  void update_params(const float_array_map& weights, bool use_quantization = false);
```

Comment:
don't forget to update the documentation for `use_quantization` in the above comment block